tests/test_output.py

```diff
@@ -0,0 +1,68 @@
+""" Tests to ensure the output_functions.py functions work properly. """
+
+import pytest
+import colorama as color
+from gatorgrade.output import output_functions
+
+color.init()
+
+
+@pytest.fixture()
+def test_output_shows_green(capsys):
+    """Test for ouput of a passed check will show green check."""
+    output_functions.output_passed_checks(passed_checks=("Remove All TODOs", True, ""))
+    out, err = capsys.readouterr()
+    expected_output = f"{color.Fore.GREEN}\u2714"
+    actual_output = out
+    assert expected_output in actual_output
+    assert err == ""
+
+
+def test_output_shows_red(capsys):
+    """Test for output of failed check to show red color using Colorama."""
+    output_functions.output_failed_checks(
+        failed_checks=("Remove all TODOs", False, "3 TODOs found in example.py")
+    )
+    out, err = capsys.readouterr()
+
+    expected_output = f"{color.Fore.RED}"
+    actual_output = out
+    assert expected_output in actual_output
+    assert err == ""
+
+
+def test_output_shows_yellow(capsys):
+    """Testing the failed check will show a yellow message in output."""
+    output_functions.output_failed_checks(
+        failed_checks=("Remove all TODOs", False, "3 TODOs found in example.py")
+    )
```

Comment:
Why do we expect this to product yellow output for the same input that we provided above that produces red output?

Reply:
This test is to prove the diagnostic will be produced for failed checks and the yellow color is properly displayed.